docs/source/getting-started/newcomer.rst

```diff
@@ -99,6 +99,10 @@ Before you start using Scancode
 
     Refer :ref:`synopsis_quickstart` to make sure you are using the scan command correctly.
 
+.. note::
+
+    For Windows, Refer to :ref:`_windows_app_install` for installing easily using Releases.
```

Comment:
@AyanSinhaMahapatra is this "_windows_app_install" a link to something else?

Reply:
IDK if you tagged the wrong guy or it was intentional but 
its a reference to the Windows Installation page section. 
@pombredanne

Reply:
@OsmiumOP yes you're right, and @pombredanne tagged me because I added this page (though quite a while ago) :P

Reply:
All good then!
Thanks